tests/integration/proxy/testproxymain.h

```diff
@@ -1,4 +1,19 @@
 #include "many-types_proxy.h"
 
-class TestProxyImpl : public org::gdbus::codegen::glibmm::TestProxy {
+class TestProxyImpl {
+public:
+    TestProxyImpl();
+
+    void fail();
+    int exit_status() const { return m_exit_status; }
+
+    sigc::signal<void> &done() { return m_done; }
+
+private:
+    void proxy_created(const Glib::RefPtr<Gio::AsyncResult> result);
+
+private:
+    Glib::RefPtr<org::gdbus::codegen::glibmm::TestProxy> m_proxy;
+    sigc::signal<void> m_done;
+    int m_exit_status = EXIT_SUCCESS;
```

Comment:
#include <cstdlib> for EXIT_SUCCESS/EXIT_FAILURE.

Reply:
Done

Reply:
Only added in `.cpp` but not here in `.h` though.

Reply:
ouch :-)

Done.